src/api/lib/tasks/dev/requests.rake

```diff
@@ -199,5 +199,315 @@ namespace :dev do
+        create_history_elements(review, params[:history_elements])
+      end
+    end
+    # rubocop:enable Metrics/CyclomaticComplexity
+    # rubocop:enable Metrics/PerceivedComplexity
+
+    def create_history_elements(review, params)
+      case review.state
+      when :accepted
+        klass = HistoryElement::ReviewAccepted
+      when :declined
+        klass = HistoryElement::ReviewDeclined
+      end
+      params.each do |param|
+        klass.create!(op_object_id: review.id, user_id: param[:user_id], comment: param[:comment])
+      end
+    end
+
+    def prepare_history_elements_data(params)
+      return [] if params.nil?
+
+      if params.is_a?(Array)
+        params.map do |param|
+          user_id = find_user(param['who']).id if param['who']
+          {
+            user_id: user_id,
+            description: param['description'],
+            comment: param['comment']
+          }
+        end
+      else
+        user_id = find_user(params['who']).id if params['who']
+        [{
+          user_id: user_id,
+          description: params['description'],
+          comment: params['comment']
+        }]
+      end
+    end
+
+    def clone_project(project_name:)
+      project = Project.find_or_create_by(name: project_name)
+      config = make_api_request(url: "#{base_api_url}/source/#{project.name}/_config")
+      clone_prj_configs(config: config, project: project, comment: "Cloned from #{project.name}")
+
+      request = make_api_request(url: "#{base_api_url}/source/#{project.name}/_meta")
+      request_data = Xmlhash.parse(request)
+
+      create_users_and_groups_from_meta(request_data: request_data)
+      create_repositories_from_meta(request_data: request_data, target_project: project)
+      clone_meta(meta: request, comment: "Cloned _meta from #{project.name}", project: project)
+    end
+
+    def clone_meta(meta:, comment:, project:)
+      params = ActionController::Parameters.new({ meta: meta, comment: comment, project: project.name })
+      meta_validator = MetaControllerService::MetaXMLValidator.new(params)
+      meta_validator.call
+      updater = MetaControllerService::ProjectUpdater.new(project: project, request_data: meta_validator.request_data).call
+      print_message(updater.errors) if updater.errors
+    end
+
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
+    def create_repositories_from_meta(request_data:, target_project:)
+      [request_data['repository']].flatten.each do |repository|
+        if repository['releasetarget'] && repository['releasetarget']['project']
+          project = Project.find_or_create_by(name: repository['releasetarget']['project'])
+          proj_repo = project.repositories.find_or_initialize_by(name: repository['releasetarget']['repository'])
+        elsif repository['path'].is_a?(Array)
+          repository['path'].each do |path|
+            project = Project.find_or_create_by(name: path['project'])
```

Comment:
It makes little sense to create local projects for this. You should use the projects over the interconnect. So 

```
  <repository name="15.5">
    <path project="openSUSE:Leap:15.5" repository="standard"/>
    <arch>x86_64</arch>
  </repository>
```

becomes

```
  <repository name="15.5">
    <path project="openSUSE.org:openSUSE:Leap:15.5" repository="standard"/>
    <arch>x86_64</arch>
  </repository>
```